src/commands/config/unset.ts

```diff
@@ -36,13 +36,18 @@ export class UnSet extends SfCommand<SetOrUnsetConfigCommandResult> {
     if (!argv || argv.length === 0) {
       throw messages.createError('error.NoConfigKeysFound');
     }
-    const config: Config = await Config.create(Config.getDefaultOptions(flags.global));
+    const config = await Config.create(Config.getDefaultOptions(flags.global));
 
     await config.read();
     for (const key of argv as string[]) {
       try {
         const resolvedName = this.configAggregator.getPropertyMeta(key)?.newKey ?? key;
         config.unset(resolvedName);
+
+        if (!flags.global && this.configAggregator.getLocation(resolvedName) === 'Global') {
+          // if the user wanted to unset a global value, and it's not in the global, warn them
```

Comment:
This comment is confusing to me, particularly the "and it's not in the global" part.  What do you think about wording it this way?
```suggestion
          // If the config var is still set globally after an unset and the user didn't have the `--global` flag set, warn them.
```